### ISSUE TITLE
ClassNotFoundException: org.apache.commons.io.output.UnsynchronizedByteArrayOutputStream

### ISSUE THREAD

**Issue by javier-godoy** [FlowingCode/GridExporterAddon#67]
I've just upgraded the demos to 24.2.0.alpha2 and export fails with

(excel)
```
java.lang.ClassNotFoundException: org.apache.commons.io.output.UnsynchronizedByteArrayOutputStream
        at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445) ~[na:na]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:587) ~[na:na]
        at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:149) ~[app.jar:1.0.0-SNAPSHOT]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[na:na]
        at org.apache.poi.poifs.filesystem.FileMagic.valueOf(FileMagic.java:209) ~[poi-5.2.3.jar!/:5.2.3]
        at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:222) ~[poi-5.2.3.jar!/:5.2.3]
        at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:185) ~[poi-5.2.3.jar!/:5.2.3]
        at com.flowingcode.vaadin.addons.gridexporter.ExcelInputStreamFactory.getBaseTemplateWorkbook(ExcelInputStreamFactory.java:289) ~[grid-exporter-addon-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT]
        at com.flowingcode.vaadin.addons.gridexporter.ExcelInputStreamFactory.createInputStream(ExcelInputStreamFactory.java:79) ~[grid-exporter-addon-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT]
        at com.vaadin.flow.server.StreamResource$Pipe.createInputStream(StreamResource.java:99) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.StreamResource$Pipe.accept(StreamResource.java:83) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.communication.StreamResourceHandler.handleRequest(StreamResourceHandler.java:86) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.communication.StreamRequestHandler.handleRequest(StreamRequestHandler.java:110) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.VaadinService.handleRequest(VaadinService.java:1529) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.VaadinServlet.service(VaadinServlet.java:398) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.spring.SpringServlet.service(SpringServlet.java:106) ~[vaadin-spring-24.2.0.alpha2.jar!/:na]
```

PDF
```
java.lang.ClassNotFoundException: org.apache.commons.io.output.UnsynchronizedByteArrayOutputStream
        at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445) ~[na:na]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:587) ~[na:na]
        at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:149) ~[app.jar:1.0.0-SNAPSHOT]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[na:na]
        at com.flowingcode.vaadin.addons.gridexporter.DocxInputStreamFactory.getBaseTemplateDoc(DocxInputStreamFactory.java:383) ~[grid-exporter-addon-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT]
        at com.flowingcode.vaadin.addons.gridexporter.DocxInputStreamFactory.createDoc(DocxInputStreamFactory.java:106) ~[grid-exporter-addon-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT]
        at com.flowingcode.vaadin.addons.gridexporter.PdfInputStreamFactory.createInputStream(PdfInputStreamFactory.java:53) ~[grid-exporter-addon-2.0.0-SNAPSHOT.jar!/:2.0.0-SNAPSHOT]
        at com.vaadin.flow.server.StreamResource$Pipe.createInputStream(StreamResource.java:99) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.StreamResource$Pipe.accept(StreamResource.java:83) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.communication.StreamResourceHandler.handleRequest(StreamResourceHandler.java:86) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.communication.StreamRequestHandler.handleRequest(StreamRequestHandler.java:110) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.VaadinService.handleRequest(VaadinService.java:1529) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.server.VaadinServlet.service(VaadinServlet.java:398) ~[flow-server-24.2.0.alpha2.jar!/:24.2.0.alpha2]
        at com.vaadin.flow.spring.SpringServlet.service(SpringServlet.java:106) ~[vaadin-spring-24.2.0.alpha2.jar!/:na]
```

**Comment by javier-godoy** [comment]
UnsynchronizedByteArrayOutputStream was added to commons-io 2.7
poi-5.2.3 requires commons-io 2.11.0 but we have a transitive dependency with commons-io 2.6 from grid-helpers

![image](https://github.com/FlowingCode/GridExporterAddon/assets/11554739/ec99a057-ae57-4d88-8a5c-8e060be592fd)

**Comment by javier-godoy** [comment]
Issue was fixed in grid-helpers 1.1.1
